app/src/main/java/com/example/itacademyhw/room/model/UserEntity.kt

```diff
@@ -0,0 +1,14 @@
+package com.example.itacademyhw.room.model
+
+import androidx.room.ColumnInfo
+import androidx.room.Entity
+import androidx.room.PrimaryKey
+
+
+@Entity
+data class UserEntity(
+    @PrimaryKey(autoGenerate = true) val id: Long = 0,
+    @ColumnInfo(name = "first_name") val firstName: String?,
+    @ColumnInfo(name = "last_name") val lastName: String?,
+    @ColumnInfo(name = "photo_name") val photoName: String = ""
+)
```

Comment:
можо было сюда положить фото, добавив поле, игнорируемое в бд
```suggestion
data class UserEntity(
    @PrimaryKey(autoGenerate = true) val id: Long = 0,
    @ColumnInfo(name = "first_name") val firstName: String?,
    @ColumnInfo(name = "last_name") val lastName: String?,
    @ColumnInfo(name = "photo_name") val photoName: String = "",
    @Ignore photo: Bitmap? = null
)
```